pipelines/resource_pipeline.go

```diff
@@ -174,7 +167,7 @@ type Pipeline struct {
 	Health               pipelines.GetPipelineResponseHealth `json:"health,omitempty"`
 	LastModified         int64                               `json:"last_modified,omitempty"`
 	LatestUpdates        []pipelines.UpdateStateInfo         `json:"latest_updates,omitempty"`
-	RunAs                pipelines.RunAs                     `json:"run_as,omitempty"`
+	RunAs                *pipelines.RunAs                    `json:"run_as,omitempty"`
```

Comment:
This needs to be a pointer to avoid serializing an empty `run_as {}` in generated terraform plans.